cob_bringup/components/ros_control_base.launch

```diff
@@ -25,10 +26,12 @@
 		<arg name="reset_errors_before_recover" value="$(arg reset_errors_before_recover)"/>
 	</include>
 
-	<include unless="$(arg sim)" file="$(find cob_omni_drive_controller)/launch/stuck_detector.launch">
-		<arg name="timeout" default="2.0"/>
-		<arg name="threshold" default="0.174533"/>
-	</include>
+	<group unless="$(arg disable_stuck_detector)">
```

Comment:
The stuck detector triggers way too fast for the old ELMOS (-gold) and heavy base. We never had issues without the stuck detector so we removed this new feature. 

We have trouble with the stuck detector on the raw3-3 as well, especially if the batteries are low.

Options are 1) disable 2) fix the stuck detector to work with the old hardware